docs/components/tooltip/README.md

```diff
@@ -158,10 +158,11 @@ CdrTooltip can also be controlled programmatically using the `open` prop. Howeve
     @blur="open = false"
     :icon-only="true"
     :with-background="true"
+    aria-label="Custom Trigger example: additional information"
     aria-describedby="tooltip-custom-example"
   >
     <template #icon>
-      <icon-information-stroke/>
+      <icon-information-stroke title="info icon"/>
```

Comment:
Our guidance is to use an `sr-only` text span rather than a title attribute. If title is needed a `<title>...</title>` can be passed into the default slot

https://rei.github.io/rei-cedar-docs/components/icon/#accessibility

Reply:
Yes that would be ok also but as the real info is communicated via the aria-label here, additional sr text seems overkill, I also question this advice as svg's have better support than they did 6 years ago

Reply:
I could add an aria-labelledby to the svg also linking to the button aria-label... there seem to be a lot of options for this